npm/private/npm_link_package_store.bzl

```diff
@@ -34,6 +35,17 @@ If unset, the package name of the src npm_package_store is used.
 If set, takes precendance over the package name in the src npm_package_store.
 """,
     ),
+    "bins": attr.string_dict(
+        doc = """Dictionary of `node_modules/.bin` binary files to create mapped to their node entry points.
```

Comment:
I think spelling out what the keys and values are here would be helpful. Or adding an sample.I wouldn't know what to put unless I saw an example.

Reply:
Added more examples